host/sgxquote.c

```diff
@@ -12,31 +13,57 @@ OE_STATIC_ASSERT(sizeof(sgx_report_t) == 432);
 
 oe_result_t oe_sgx_qe_get_target_info(uint8_t* target_info)
 {
+    oe_result_t result = OE_OK;
     quote3_error_t err =
         sgx_qe_get_target_info((sgx_target_info_t*)target_info);
-    return (err == SGX_QL_SUCCESS) ? OE_OK : OE_PLATFORM_ERROR;
+
+    if (err != SGX_QL_SUCCESS)
+    {
+        OE_TRACE_ERROR("quote3_error_t=0x%x\n", err);
+        OE_RAISE(OE_PLATFORM_ERROR);
+    }
+
+done:
+    return result;
 }
 
 oe_result_t oe_sgx_qe_get_quote_size(size_t* quote_size)
 {
+    oe_result_t result = OE_OK;
```

Comment:
Initialize to OE_FAILURE or OE_UNEXPECTED (confirm with @mikbras)

Reply:
Other quote related APIs are using OE_FAILURE, I don;t see  any reason to use other error code.